pkg/checks/checks.go

```diff
@@ -188,6 +288,44 @@ out:
 		}
 	}
 
+	// generate a list of the expected static pods
+	pods := make(map[string]struct{ Namespace string })
+	for _, app := range staticWhitelist {
+		switch app.NodeType {
+		case "all":
+			for _, nodes := range nodes {
+				for _, node := range nodes {
+					pods[fmt.Sprintf("%s-%s", app.Name, node.Name)] = struct{ Namespace string }{Namespace: app.Namespace}
+				}
+			}
+		default:
+			for _, node := range nodes[app.NodeType] {
+				pods[fmt.Sprintf("%s-%s", app.Name, node.Name)] = struct{ Namespace string }{Namespace: app.Namespace}
+			}
+		}
+	}
+
+	// static pods
+	for podname, podinfo := range pods {
+		log.Info(fmt.Sprintf("checking         static: [%s] %s", podname, podinfo.Namespace))
```

Comment:
lots of whitespace

Reply:
Also log.Infof instead of log.Info+fmt.Sprintf

Reply:
This was my attempt at aligning the output strings.  This makes the output appear uniform:
```
INFO[2018-09-10T12:27:07-04:00] checking infrastructure service health        resourceGroup=kwoodsontest31
INFO[2018-09-10T12:27:07-04:00] checking   daemonset[0]: [openshift-metrics] prometheus-node-exporter  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:28:55-04:00] checking   daemonset[1]: [openshift-node] sync  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:28:55-04:00] checking   daemonset[2]: [openshift-sdn] ovs  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:28:55-04:00] checking   daemonset[3]: [openshift-sdn] sdn  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:28:55-04:00] checking statefulset[0]: [openshift-metrics] prometheus  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[0]: [default] docker-registry  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[1]: [default] router     resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[2]: [default] registry-console  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[3]: [kube-service-catalog] apiserver  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[4]: [kube-service-catalog] controller-manager  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[5]: [openshift-infra] bootstrap-autoapprover  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[6]: [openshift-ansible-service-broker] asb  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[7]: [openshift-template-service-broker] apiserver  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[8]: [openshift-infra] bootstrap-autoapprover  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking  deployment[9]: [openshift-web-console] webconsole  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking         static: [controllers-master-000002] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking         static: [etcd-master-000000] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking         static: [etcd-master-000001] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking         static: [etcd-master-000002] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:47-04:00] checking         static: [logbridge-infra-000001] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [logbridge-master-000002] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [api-master-000001] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [logbridge-infra-000000] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [api-master-000002] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [controllers-master-000000] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [logbridge-master-000000] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [api-master-000000] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [logbridge-compute-000000] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [logbridge-master-000001] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] checking         static: [controllers-master-000001] kube-system  resourceGroup=kwoodsontest31
INFO[2018-09-10T12:31:48-04:00] done checking infrastructure service health   resourceGroup=kwoodsontest31
```

Reply:
Don't do this.  In the RP this is logged out as json and searchability is more important than justifying.